### PR TITLE
feat: add layout/main component

### PR DESCRIPTION
**Pull request by rubenmantilladev**
A new route module (LayoutModule) was created along with the "LayoutComponent" component, in which the <router-outlet> and <app-header> component were added. The addition of the <app-footer> (waiting for the component) is still pending.

Completed tasks:
- [x] Crear el componente
- [x] Agregar el router-outlet al HTML
- [x] Agregar la ruta base al archivo de rutas

### CODE REVIEW

**Comment by rubenmantilladev** [comment]
[Crear el componente Main](https://trello.com/c/wcaNnd3E/15-crear-el-componente-main)

File: src/app/layout/layout.component.html
Comment (ivanfdp) [inline src/app/layout/layout.component.html:1]: // TODO: add footer tag

File: src/app/app-routing.module.ts
Comment (ivanfdp) [inline src/app/app-routing.module.ts:1]: La ruta default se puede cambiar ya al default

File: src/app/layout/layout-routing.module.ts
Comment (ivanfdp) [inline src/app/layout/layout-routing.module.ts:1]: // TODO  routes using header and footer components here at children's routes

**Review by ivanfdp — APPROVED** [review]

**Pull request merged**